shell/browser/electron_api_ipc_handler_impl.cc

```diff
@@ -48,16 +58,27 @@ void ElectronApiIPCHandlerImpl::Invoke(bool internal,
                                        const std::string& channel,
                                        blink::CloneableMessage arguments,
                                        InvokeCallback callback) {
+  electron::crash_keys::SetCrashKey("electron.ipc.method", "invoke");
+  electron::crash_keys::SetCrashKey("electron.ipc.channel", channel);
```

Comment:
It would it be better to move these keys inside the condition to validate the crash.

```suggestion
if (api_web_contents) {
} else {
  electron::crash_keys::SetCrashKey("electron.ipc.method", "invoke");
  electron::crash_keys::SetCrashKey("electron.ipc.channel", channel);
 }
```

Reply:
I was thinking that by always setting the crash keys here, they would be included for any crash involving handling of that IPC. I could be misunderstanding crash key usage though.

Reply:
Yes if we want to know the details of those keys in all cases that would be the place to set, but for this particular case it would be good to validate that `api_web_contents == nullptr` when the crash happens hence I suggested the special casing. You can always add a top level crash key that does `electron::crash_keys::SetCrashKey("api::webContents is null", api_web_contents == nullptr);` as an alternative.